app/util.js

```diff
@@ -123,5 +125,50 @@ utils.validatePackage = function(packageName) {
     }
     return true;
 }
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+utils.validateCamelVersion2 = async function (version, dsl) {
+    const artifactId = "camel:spring";
```

Comment:
camel-spring?

Reply:
Just for a test for now - I will change that based on the inputs the user provides (i.e. which dsl to look for)